index.js

```diff
@@ -1,11 +1,18 @@
-let enabled =
-  !("NO_COLOR" in process.env) &&
-  ("FORCE_COLOR" in process.env ||
-    process.platform === "win32" ||
-    (process.stdout != null &&
-      process.stdout.isTTY &&
-      process.env.TERM &&
-      process.env.TERM !== "dumb"))
+const {env} = process;
+
+const isDisabled = "NO_COLOR" in env
+const isForced = "FORCE_COLOR" in env
+const isWindows = process.platform === "win32"
+
+const isCompatibleTerminal = (process.stdout != null &&
+    process.stdout.isTTY &&
+    env.TERM &&
+    env.TERM !== "dumb")
+
+const isCI = ('CI' in env && ['CIRCLECI', 'GITLAB_CI', 'GITHUB_ACTIONS'].some(sign => sign in env))
+
+let enabled = (!isDisabled) &&
+  (isForced || isWindows || isCompatibleTerminal || isCI)
```

Comment:
Can we do without the destructuring assignment (I know it's a bit strange, but we technically still support pre Node 6) and format the expression to this:

```js
const env = process.env

const isDisabled = "NO_COLOR" in env
const isForced = "FORCE_COLOR" in env
const isWindows = process.platform === "win32"

const isCompatibleTerminal =
  process.stdout != null &&
  process.stdout.isTTY &&
  env.TERM &&
  env.TERM !== "dumb"

  const isCI =
  "CI" in env &&
  ("GITHUB_ACTIONS" in env || "GITLAB_CI" in env || "CIRCLECI" in env)

let enabled =
  !isDisabled && (isForced || isWindows || isCompatibleTerminal || isCI)
```

Reply:
Done!

Reply:
Thank you!